src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -198,4 +215,11 @@ object UserService {
 
   def genWorkbenchUserId(currentMilli: Long): WorkbenchUserId =
     WorkbenchUserId(genRandom(currentMilli))
+
+  def validateEmailAddress(email: WorkbenchEmail) = {
+    email.value match {
```

Comment:
If the user entered some leading/trailing whitespace, can we just lop that off for them rather than throwing an error?